octoprint_mrbeam/iobeam/iobeam_handler.py

```diff
@@ -1068,7 +1069,13 @@ def _handle_exhaust(self, dataset):
         :param dataset:
         :return: error count
         """
-        # self._logger.info("exhaust dataset: '%s'", dataset)
+        self._logger.debug("exhaust dataset: '%s'", dataset)
+        # get the pressure sensor reading this will come as dust with the current iobeam version
+        if "dust" in dataset:
+            vals = {
```

Comment:
not a good name

Reply:
for what?

Reply:
I guess he was referring to vals. we could call it pressure_values 🤔

Reply:
I guess he was referring to vals. we could call it pressure_values 🤔